.github/workflows/update-prod-env.yml

```diff
@@ -0,0 +1,28 @@
+name: Update Prod Env
+
+permissions:
+  id-token: write
+  contents: read
+  packages: write
+
+on:
+  workflow_dispatch:
+
+jobs:
+  copy-tag:
+    runs-on: depot-ubuntu-22.04
+
+    steps:
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup crane
+        uses: imjasonh/setup-crane@v0.4
+
+      - name: Copy to prod tag
+        run: |
+          crane tag ghcr.io/${{ github.repository }}-enterprise:main prod
```

Comment:
do we want to keep doing this, or do we want to start actually promoting stuff from test?

Reply:
I figured, for now, we do it consistently, and we can change them all later.